extensions/ql-vscode/src/remote-queries/view/RepositoriesSearch.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { ChangeEvent } from 'react';
+import { TextInput } from '@primer/react';
+import { SearchIcon } from '@primer/octicons-react';
+
+interface RepositoriesSearchProps {
+  filterValue: string;
+  setFilterValue: (value: string) => void;
+}
+
+const RepositoriesSearch = ({ filterValue, setFilterValue }: RepositoriesSearchProps) => {
+  return <>
+    <TextInput
+      block
+      sx={{
```

Comment:
This doesn't look like it's needed. We what it looks like without it and it looks fine in both modes to us.

Reply:
Hmm yeah I have no idea what it should look like in red 😂 we just checked light and dark mode. Ideally we'd be setting colors through the Primer theme but that's something to look into separately!

Reply:
Handling all kinds of themes will be very, very tricky. But, I think we should at least make a best effort for the common themes (though, I don't know if red is very common).